deploy/conf/production-gs/site.conf

```diff
@@ -24,11 +24,11 @@ lucuma-sso {
 }
 
 web-server {
-  port = 9090
-  insecure-port = 9091
+  port = 443
```

Comment:
This may not work. for por 443 you need root rights, or perhaps docker has them?
My usual practice has been (non docker apps) to use a non privileged port and instead do a port mapping via e.g.. iptables